moving doc to the platform section (#9710)
supabase/supabase: apps/reference/docs/guides/platform/custom-domains.mdx

---
id: custom-domains
title: Custom Domains
description: Configuring a Custom Domain for your Supabase project.
---

import Tabs from '@theme/Tabs'
import TabItem from '@theme/TabItem'

:::note

Custom Domains are currently in beta, and are being slowly made available to projects.

:::

Custom domains allow you to present a branded experience to your users. You can also use a subdomain for the purposes of this guide.

> To get configure a custom domain, you must first:
> 
> - [Install](/docs/guides/cli) the Supabase CLI 1.10.1+
> - [Login](/docs/guides/cli/local-development#log-in-to-the-supabase-cli) to your account usig the CLI
> - Enable the Custom Domains add-on in the Billing page of the Dashboard.

Reviewer: Can we add a link to enable the add-on? Would it be under https://app.supabase.com/project/_/settings/billing?
```suggestion
To get started:

1. [Install](/docs/guides/cli) the Supabase CLI 1.10.1+.
1. [Log in](/docs/guides/cli/local-development#log-in-to-the-supabase-cli) to your Supabase account using the CLI.
1. Enable the Custom Domains add-on in the Billing page of the Dashboard.
```

## Configure a Custom Domain

> In this example we will assume your Supabase project is `foobar` with a corresponding API URL `foobar.supabase.co`. We'll step you through setting up your own subdomain "api.example.com".

Reviewer: ```suggestion
This example assumes your Supabase project is `foobar` with a corresponding API URL `foobar.supabase.co` and configures a custom domain at `api.example.com`.
```
